docker/postgres/docker-entrypoint-initdb.d/mock-data/interfaith_DDL.sql

```diff
@@ -54,22 +50,25 @@ CREATE TABLE needs (
   need_active BOOLEAN NOT NULL
 );
 
-DROP TABLE IF EXISTS needcallouts cascade;
-CREATE TABLE needcallouts (
-  needcallout_id BIGSERIAL PRIMARY KEY,
-  needcallout_quantity INT NOT NULL,
+CREATE TABLE calloutneeds (
+  calloutneed_id BIGSERIAL PRIMARY KEY,
+  calloutneed_quantity INT NOT NULL,
   need_id BIGINT REFERENCES needs,
   callout_id BIGINT REFERENCES callouts,
-  needcallout_active BOOLEAN NOT NULL
+  calloutneed_active BOOLEAN NOT NULL
 );
 
-DROP TABLE IF EXISTS donations cascade;
 CREATE TABLE donations (
   donation_id BIGSERIAL PRIMARY KEY,
   donation_quantity INT NOT NULL,
   donation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-  needcallout_id BIGINT REFERENCES users NOT NULL,
+  calloutneed_id BIGINT REFERENCES users NOT NULL,
   -- donor_id BIGINT REFERENCES donors NOT NULL
 );
 
--- transaction stuff, activations, 
+CREATE TABLE alerts (
+  alert_id BIGSERIAL PRIMARY KEY,
+  alert_sent_date TIMESTAMP NULL,
+  alert_created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
```

Comment:
Is created_date a necessary field? It obviously doesn't hurt us to capture it so no worries either way just curious.

Another possible field: alert_text/description? So she can say "Click on the app now to check out what we need for this winter" or something along those lines? Could always add it now and remove the field later, which seems easier than trying to add it later?

Reply:
Created date is not necessary, it's just really easy to implement and gives us more information about an alert.

Good call on the alert_text / alert_description! I can get that added.